lib/dom-parser.js

```diff
@@ -1,8 +1,14 @@
 var conventions = require("./conventions");
 var entities = require('./entities');
 
+var DOMImplementation = require('./dom').DOMImplementation;
+var sax = require('./sax');
```

Comment:
nit: I would love to see all require statements in one block and ordered alphabetically.

```suggestion
var dom = require('./dom')
var entities = require('./entities');
var sax = require('./sax');

var DOMImplementation = dom.DOMImplementation;
```